GoogleForCreators/web-stories-wp
# Dashboard: Added base animations to Wellbeing template

## Summary

This PR adds base animations to the Wellbeing template.

## To-do

- ~I still need to update our missing animation parts spreadsheet to reflect the changes for Wellbeing.~

## User-facing changes

None.

## Testing Instructions

0.) Enable animations by setting enableAnimations to true in Dashboard.php.
1.) Go to "Explore Templates".
2.) Click view on "DIY".
3.) Click on each page of the DIY template to see each one animate as they load.

---



Fixes #2212

## Screenshots

Page 1:
<img src="https://user-images.githubusercontent.com/40646372/87584231-0767f600-c692-11ea-8195-fefc4d9128b0.gif" height="300">

Page 2:
<img src="https://user-images.githubusercontent.com/40646372/87584243-0afb7d00-c692-11ea-8a5d-d5face8e69a9.gif" height="300">

Page 3:
<img src="https://user-images.githubusercontent.com/40646372/87584251-0fc03100-c692-11ea-8304-436780382f96.gif" height="300">

Page 4:
<img src="https://user-images.githubusercontent.com/40646372/87584258-12228b00-c692-11ea-92a2-6eb585ac9d70.gif" height="300">

Page 5:
<img src="https://user-images.githubusercontent.com/40646372/87584268-164ea880-c692-11ea-9ad0-ee35fbdef521.gif" height="300">

Page 6:
<img src="https://user-images.githubusercontent.com/40646372/87584276-19499900-c692-11ea-9437-ab58a457f294.gif" height="300">

Page 7:
<img src="https://user-images.githubusercontent.com/40646372/87584284-1c448980-c692-11ea-8847-1c5c9a86762e.gif" height="300">

Page 8:
<img src="https://user-images.githubusercontent.com/40646372/87584292-1f3f7a00-c692-11ea-9482-9cd5989f3d47.gif" height="300">

Page 9:
<img src="https://user-images.githubusercontent.com/40646372/87584302-236b9780-c692-11ea-9d8a-c1e57c9befd7.gif" height="300">

Page 10:
<img src="https://user-images.githubusercontent.com/40646372/87584309-26668800-c692-11ea-920f-314d2cbf31e7.gif" height="300">
